src/components/ConnectWalletButton.tsx

```diff
@@ -21,12 +26,52 @@ const ConnectWalletButton = ({ chainId }: { chainId: ChainId }) => {
 
   const pk = wallet?.getAddress();
 
+  const getIsSanctioned = async (addr?: string) => {
+    const key = process.env.REACT_APP_TRM_API_KEY;
+    if (addr && key) {
+      const resp = await fetch(
+        `https://api.trmlabs.com/public/v1/sanctions/screening`,
+        {
+          method: "POST",
+          headers: {
+            "Content-Type": "application/json",
+            Authorization:
+              "Basic " + Buffer.from(`${key}:${key}`).toString("base64"),
+          },
+          body: JSON.stringify([
+            {
+              address: addr,
+              // address: "149w62rY42aZBox8fGcmqNsXUzSStKeq8C", // sanctioned address example
```

Comment:
Should we remove this comment?

Reply:
I think it's useful to check quickly if service is working all right 🙌🏼